make/harbor.yml.tmpl

```diff
@@ -246,7 +246,16 @@ upload_purging:
   interval: 24h
   dryrun: false
 
-# Cache related config
+# cache layer configurations
+# if this feature enabled, harbor will cache the resource
+# `project/project_metadata/repository/artifact/manifest` in the redis
+# which especially help to improve the performance of high concurrent pulling manifest.
+# NOTICE
+# Please ensure all the harbor instances with same behaviour, all enable cache
```

Comment:
If you are deploying Harbor in HA mode, make sure that all Harbor instances have the same behavior, all with caching enabled or disabled, otherwise it can lead to potential data inconsistencies.

Reply:
updated, thanks~